src/components/Map/Icon.js

```diff
@@ -0,0 +1,9 @@
+import L from 'leaflet';
```

Comment:
Is there any way to import this with a more descriptive name? Something like "leaflet"

Reply:
I have tried to change this constant to a more descriptive one but I always get an error.

Reply:
ok, I guess we'll leave it this way then.